### PR TITLE
[Python-SDK] Docs generation

### PR DESCRIPTION
**Pull request by bbonenfant**
[INT-898]

[INT-898]: https://pachyderm.atlassian.net/browse/INT-898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: python-sdk/pachyderm_sdk/api/pfs/_additions.py [hunk 1]
```diff
@@ -137,6 +154,13 @@ def _File_from_uri(uri: str) -> File:
 
 
 def _File_as_uri(self: "File") -> str:
+    """Returns the URI for the Commit object in one of the following formats:
```
Comment (smalyala) [inline python-sdk/pachyderm_sdk/api/pfs/_additions.py:157]: "Commit" -> "File"

**Review by lbliii — APPROVED** [review]
lgtm

File: python-sdk/pachyderm_sdk/api/transaction/extension.py [hunk 2]
```diff
@@ -23,13 +24,16 @@ def __init__(
         super().__init__(channel=channel)
 
     def start_transaction(self) -> "Transaction":
-        # TODO: Should we do this?
+        """Starts a transaction and sets the transaction ID within the client.
+        This will make all subsequent resource operations performed by the client
+          occur within the returned transaction, until the transaction is finished.
```
Comment (smalyala) [inline python-sdk/pachyderm_sdk/api/transaction/extension.py:29]: Unindent

**Review by smalyala — APPROVED** [review]
LGTM

**Pull request merged**